src/components/Layout/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Styled from './styled';
+
+function Layout({ children }) {
+  return (
+    <Styled.Container className="container" data-testid="layout">
```

Comment:
Use `data-test-id` for more complex components that are harder to reach. The `Styled.Container` would be rendered as a `main` HTML and role - [check](https://www.w3.org/TR/html-aria/#docconformance)